pywinauto/unittests/test_application.py

```diff
@@ -886,6 +886,29 @@ def test_print_control_identifiers(self):
         self.dlgspec.print_control_identifiers()
         self.ctrlspec.print_control_identifiers()
 
+    def test_print_control_identifiers_file_output(self):
+        """Make sure print_control_identifiers() creates correct file"""
+        output_filename = "test_print_control_identifiers.txt"
+        self.dlgspec.print_ctrl_ids(filename=output_filename)
+        if os.path.isfile(output_filename):
+            with open(output_filename, "r") as test_log_file:
+                content = str(test_log_file.readlines())
+                self.assertTrue(content.find("['Untitled - NotepadEdit', 'Edit']") != -1)

```

Comment:
The test fails here on AppVeyor:
```python
Traceback (most recent call last):
  File "C:\projects\pywinauto\pywinauto\unittests\test_application.py", line 896, in test_print_control_identifiers_file_output
    self.assertTrue(content.find("['Untitled - NotepadEdit', 'Edit']") != -1)
AssertionError: False is not true
```

BTW using `find` function is an old ugly style from 90s. Use operator `in`.